week-11-front-end-frameworks-and-mobile/day-02/drill/math-fundamentals.md

```diff
@@ -0,0 +1,350 @@
+# <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Interview Workshop: Math Fundamentals
+
+## Math Terminology
+
+**Here are some mathematical terms you need to know, but it is by no means an exhaustive list.**
+
+<table>
+  <tr>
+    <th>Term</th>
+    <th>Definition</th>
+  </tr>
+  <tr>
+    <td><strong>Absolute Value</strong></td>
+    <td>The magnitude of a number. It is the number with the sign (+ or -) removed and is symbolized using two vertical straight lines (|5|). Also called modulus.</td>
```

Comment:
Is this really also called a `modulus`? If it is, that's cool, but I thought `modulus` was the `%` that calculates the remainder. If it is a term for both, I would make that clear.